jekyll/__glossary/context.md

```diff
@@ -0,0 +1,5 @@
+---
+term: Context
+---
+
+Contexts provide a mechanism for securing and sharing environment variables across projects. The environment variables are defined as name/value pairs and are injected at runtime. To use environment variables set on the Contexts page, the user running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. See the [Contexts]({{ site.baseurl }}/2.0/contexts/) document for instructions.
```

Comment:
Third sentence is a little long. I'd separate the notion of "setting environment variables" from using them:

"Environment variables are set on the Contexts page. To use these variables in a workflow, users must be members of the context's organization. The rule must allow access to all projects in the org."

Question: why do we explicitly call out a link to another document as "the X document"?

Reply:
fixed, thanks. Links to other documents should be called out by the title to help people know they are going to another document, what that document contains, and to orient them when they get there.